app/pods/publications/overview/_base/controller.js

```diff
@@ -0,0 +1,103 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import tableColumns from 'frontend-kaleidos/config/publications/overview-table-columns';
+
+export default class PublicationsOverviewBaseController extends Controller {
+  /** @abstract @type {string[]} */
+  @tracked defaultColumns = [];
+
+  /** @abstract @type {string} */
+  @tracked routeName = 'base';
```

Comment:
Dit begrijp ik niet. Het werkt, maar het lijkt me niet vanzelfsprekend dat deze `@tracked` niet dient voor tracking, maar om het field voor de uitvoering van `initColumnsDisplayConfig` te setten.
Mijn alternatief op de deprecated `init`-hook was om in de constructor van de concrete controller iets te doen, vb. een eigen hook `super.initializeController()` (of meteen `super.initColumnsDisplayConfig()`).
Anders zou ik er toch een comment bij plaatsen.